src/plugins/index.ts

```diff
@@ -0,0 +1,73 @@
+import type { CAIP19 } from '@shapeshiftoss/caip'
+import { FeatureFlag } from 'constants/FeatureFlag'
+
+import { Route } from '../Routes/helpers'
+
+const activePlugins = ['bitcoin']
```

Comment:
this seems like it would go better as an argument to `PluginManager.create()`

Reply:
no such function exists.